### ISSUE TITLE
input type=search should have default ARIA role=searchbox

### ISSUE THREAD

**Issue by jasonkiss** [w3c/html#488]
The [searchbox role](http://w3c.github.io/aria/aria/aria.html#searchbox) is ARIA 1.1, but the appropriate default role for [input type=search](http://w3c.github.io/html/sec-forms.html#text-typetext-state-and-search-state-typesearch), and should be an allowed role for input type=text.

**Comment by stevefaulkner** [comment]
@jasonkiss the non-normative advice in HTML5 for input type=search is as you describe:

> textbox, searchbox with no list attribute (default - do not set) or with a list attribute: combobox (default - do not set). 

please [file a bug](https://github.com/w3c/html-aria/issues) on aria in html so it reflects same.
